homeassistant/components/apcupsd/__init__.py

```diff
@@ -23,52 +20,6 @@
 MIN_TIME_BETWEEN_UPDATES: Final = timedelta(seconds=60)
 
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We need to keep this around to prevent configuration checks to fail

```py
CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
```

Reply:
Sure! Added this back. Do we intend to keep this on a longer term, or do we plan on removing this in the near future?

Reply:
In general tend to keep it around for a long time